aurora-architecture-overview.md

```diff
@@ -0,0 +1,249 @@
+# Aurora Dashboard – Architecture Overview
+
+## Introduction
+
+Aurora Dashboard is a modern replacement for the legacy OpenStack Horizon interface. It is designed to be fast, modular, and developer-friendly, offering a clean and type-safe experience across the stack. By following a Backend-for-Frontend (BFF) architecture, we abstract away OpenStack complexity and deliver a UI that is decoupled, composable, and scalable.
```

Comment:
I guess Horizon is not officially legacy? (it is our opinion)

Reply:
:) Ups

Reply:
WDYT should I replace it with Elektra or just remove this phrase ?